com.unity.perception/Runtime/GroundTruth/PerceptionCamera_InstanceSegmentation.cs

```diff
@@ -43,17 +57,31 @@ void SetupInstanceSegmentation()
             var customPassVolume = this.GetComponent<CustomPassVolume>() ?? gameObject.AddComponent<CustomPassVolume>();
             customPassVolume.injectionPoint = CustomPassInjectionPoint.BeforeRendering;
             customPassVolume.isGlobal = true;
-            var instanceSegmentationPass = new InstanceSegmentationPass()
+            m_InstanceSegmentationPass = new InstanceSegmentationPass()
             {
                 name = "Instance segmentation pass",
                 targetCamera = GetComponent<Camera>(),
                 targetTexture = m_InstanceSegmentationTexture
             };
-            instanceSegmentationPass.EnsureInit();
-            customPassVolume.customPasses.Add(instanceSegmentationPass);
-#endif
-#if URP_PRESENT
-            AddScriptableRenderPass(new InstanceSegmentationUrpPass(myCamera, m_InstanceSegmentationTexture));
+            m_InstanceSegmentationPass.EnsureInit();
+            customPassVolume.customPasses.Add(m_InstanceSegmentationPass);
+
+            // TODO: Note - the implementation here differs substantially from how things are done in SemanticSegmentationLalber
+            // Also, the naming convention doesn't line up, shouldn't instance segmentation be a labeller?  At least in
+            // architecture
+            m_LensDistortionPass = new LensDistortionPass(GetComponent<Camera>(), m_InstanceSegmentationTexture)
```

Comment:
Which part of the implementation is different?
We decided to keep instance segmentation itself out of a labeler for now because we didn't want to have to deal with dependencies between labelers. Also, there is no configuration of the instance segmentation logic, so there is no point in having more than one. At some point we'll have an instance segmentation labeler though to save the images to disk, but perhaps not before the new architecture :)

Reply:
Yeah - I'll back off this point for now, and defer to hashing this out in arch!